allensdk/brain_observatory/behavior/metadata_processing.py

```diff
@@ -57,23 +58,53 @@ def get_expt_description(session_type: str) -> str:
 
 def get_task_parameters(data):
```

Comment:
I feel like it would be good to have a docstring that describes a bit more what the data should look like:

e.g. `Nested dictionary with at least: ["items"]["behavior"]["stimuli"]`

Could you also add at least some simple typing for the inputs and returns?

Reply:
I just pushed as much of a docstring as I felt I could. Anything more, I think, requires detailed documentation of the contents of the pickle files.